src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentTwigExtension.php

```diff
@@ -73,13 +74,21 @@ public function getFunctions()
      */
     public function load($uuid)
     {
-        $contentStructure = $this->contentMapper->load(
-            $uuid,
-            $this->requestAnalyzer->getWebspace()->getKey(),
-            $this->requestAnalyzer->getCurrentLocalization()->getLocale()
-        );
+        if (!$uuid) {
+            return;
+        }
+
+        try {
+            $contentStructure = $this->contentMapper->load(
+                $uuid,
+                $this->requestAnalyzer->getWebspace()->getKey(),
+                $this->requestAnalyzer->getCurrentLocalization()->getLocale()
+            );
 
-        return $this->structureResolver->resolve($contentStructure);
+            return $this->structureResolver->resolve($contentStructure);
+        } catch (DocumentNotFoundException $e) {
+            return;
```

Comment:
We've talked about that today, and we thought it might make sense to throw some kind of `TwigException` here, or however that is called. The ones that twig throws in develop mode, and catches in production (because twig is running in strict mode). In the best case Twig would even write the error to the log. If all these assumptions are true, I would go for this way 😃

Reply:
Hm, I'm not sure which exception you mean. :) Currently it is already being catched by Twig and converted to a Twig_Runtime exception. On production this will show a HTTP 500 Error.

![image](https://user-images.githubusercontent.com/5249624/33210683-7b8ffc9a-d11b-11e7-8c68-920e1b1c9391.png)